components/ActivityIndicator/BUILD

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017-present The Material Foundation Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_ios_warnings//:strict_warnings_objc_library.bzl", "strict_warnings_objc_library")
+load("@build_bazel_rules_apple//apple:swift.bzl", "swift_library")
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_unit_test")
+
+licenses(["notice"])  # Apache 2.0
+
+strict_warnings_objc_library(
+    name = "ActivityIndicator",
+    srcs = glob([
+        "src/*.m",
+        "src/private/*.m",
+    ]),
+    hdrs = glob([
+        "src/*.h",
+        "src/private/*.h",
+    ]),
+    deps = [
+      "//components/Palettes",
+      "//components/private/Application",
+      "@material_internationalization_ios//:MDFInternationalization",
+    ],
+    enable_modules = 1,
+    includes = ["src"],
```

Comment:
Does this automatically pick-up `src/private`?

Reply:
`includes` defines the headers that clients using this dependency will have access to, so we intentionally only reveal the `src` directory.

Reply:
Ack.